tests/src/Functional/DeveloperAppFieldTest.php

```diff
@@ -545,6 +546,7 @@ protected function assertFieldVisibleOnEntityForm(string $field_label, bool $vis
    */
   protected function assertFieldVisibleOnEntityDisplay(string $app_name, string $field_label, string $field_value, bool $visible = TRUE) {
     $this->drupalGet("/user/{$this->account->id()}/apps/{$app_name}");
+    $this->assertEquals(Response::HTTP_OK, $this->getSession()->getStatusCode());
```

Comment:
Why is this necessary?

Reply:
When the test fails here for some reason, I get a more meaningful message (e.g. 404 instead of 200), and not an "element xpath=blabla not found on page".

Reply:
Ok.